### PR TITLE
(1-c) Result page #4-BaharBazargan

### PR DESCRIPTION
**Pull request by baharbazargan62**
## Description

I have made the result page which includes table that received all information from the Json file and prepare table for filtering data which came from pre page 's data

## Related to

(1-c) Result page #8

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by EneVladMihai** [comment]
Could you double check that the linked in the **Related to** points to the correct issue?

**Review by KptCnyt — APPROVED** [review]
All great, thank you

**Comment by baharbazargan62** [comment]
> Could you double check that the linked in the **Related to** points to the correct issue?

YES,I sort it out.

File: client/src/pages/result/Result.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      setLocation(e.target.value);
+    }
+  return (
+    <>
+    <Form>
+      <Form.Group className="select-group" controlId="service">
+        <Form.Label>Service</Form.Label>
+        <Form.Select aria-label="Service" value={service} onChange={selectService}>
+          {[...new Set(data.map((item) => item.Themes))].map((item,index)=>(
+            <option  key={index}>{item}</option>
```
Comment (EneVladMihai) [inline client/src/pages/result/Result.js:31]: Very nice use of `Set`!
The `key` should be something that uniquely identifies the object. While the index is sometimes a viable option if there's no other in this case we can use the `item` itself.

File: client/src/pages/result/Result.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.table{
+    width: 60%;
+    margin: 0 auto; 
+     
+}
+Button{
```
Comment (EneVladMihai) [inline client/src/pages/result/Result.css:6]: Is this used? If not better to remove as it will make the code harder to read in the future

**Review by EneVladMihai — CHANGES_REQUESTED** [review]
Very minor things. Nice work, the compoent and filtering are very cleanly implemented.

File: client/src/pages/result/Result.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          {[...new Set(data.map((item) => item.Themes))].map((item)=>(
+            <option  key={item}>{item}</option>
+          ))}
+        </Form.Select>
+      </Form.Group>
+      <Form.Group className="select-group" controlId="location">
+        <Form.Label>Location</Form.Label>
+        <Form.Select aria-label="Location" value={location} onChange={selectLocation}>
+          {[...new Set(data.map((item) => item.Zone))].map((item,index)=>(
+            <option key={index}>{item}</option>
```
Comment (EneVladMihai) [inline client/src/pages/result/Result.js:39]: Same thing about the `key` here

Comment (baharbazargan62) [reply client/src/pages/result/Result.js]: It has done.

**Review by EneVladMihai — APPROVED** [review]

**Pull request merged**